### PR TITLE
export sync-seconds to control resource resync frequency

### PR DESCRIPTION
**Pull request by shiyan2016**
### Ⅰ. Describe what this PR does
export sync-seconds to control resource resync frequency

### CODE REVIEW

**Comment by shiyan2016** [comment]
@dco

File: main.go [hunk 1]
```diff
@@ -129,6 +132,7 @@ func main() {
 		LeaderElectionNamespace:    leaderElectionNamespace,
 		LeaderElectionResourceLock: resourcelock.ConfigMapsResourceLock,
 		Namespace:                  namespace,
+		SyncPeriod:                 &syncPeriod,
```
Comment (FillZpp) [inline main.go:135]: The type of `SyncPeriod` is `*time.Duration`, which means the default value `36000` is `36us`...

How about:
```golang
        var syncPeriodStr string
	flag.StringVar(&syncPeriodStr, "sync-period", "", "Determines the minimum frequency at which watched resources are reconciled.")

	var syncPeriod *time.Duration
	if syncPeriodStr != "" {
		d, err := time.ParseDuration(syncPeriodStr)
		if err != nil {
			setupLog.Error(err, "invalid sync period flag")
		}
		syncPeriod = &d
	}

            // ...
            SyncPeriod:                 syncPeriod,
```

Then you can set the flag as `--sync-perioid=10h` or `--sync-perioid=0` to disable resync.

Comment (shiyan2016) [reply main.go]: done

**Review by FillZpp — APPROVED** [review]
/lgtm

**Pull request merged**